Add tango:// scheme to full name

mrosanes opened the pull request:
For sardana elements in sardana instruments, the tango scheme
has to be indicated in the sardana elements.

Add tango:// as scheme for sardana elements included in
the instruments. This allows to use the instruments in
taurus panels.

cpascual commented:
Not sure if this is the approach we want...

Maybe the `DEVICE_STATE_PALETTE`  should map (collapse) the Tango DevState colors into TaurusDevStateColors (as discussed in https://sf.net/p/tauruslib/tickets/236/)

cpascual commented:
I pushed a different alternative. (peer reviewed by @cmft )

Commit:
Add tango:// scheme to full name

For sardana elements in sardana instruments, the tango scheme
has to be indicated in the sardana elements.

Add tango:// as scheme for sardana elements included in
the instruments. This allows to use the instruments in
taurus panels.
